### PR TITLE
Bookstore Create and Remove books

### PR DESCRIPTION
**Pull request by ramiaquista**
## What I have done 🔨  :
- Added logic inside redux/books.js to add and remove books from Store.
- Added store configuration inside configureStore.js file.
- Added `<Provider store={store}>` inside App.js file.
- Added `uuid` import to generate unique ids for the books.
- Added logic to remove book inside Book() component.
- Added useSelector() to render data from the redux store inside the BookPage component.
- Added logic to `set` and `get` LocalStorage inside the ConfigureStore.js file
- Added styling to form inputs.

### CODE REVIEW

File: src/redux/cofigureStore.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { createStore, applyMiddleware } from 'redux';
```
Comment (helman101) [inline src/redux/cofigureStore.js]: - [x] Please import and use `combineReducers` even if you have just one reducer, as it is a project requirement 🚀

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🔴 

Hi @ramiaquista 👋, 

Nice job so far with this project 😄, but there are still some changes to do.

## Required Changes ♻️

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/cofigureStore.js (hunk 1 above)
Comment (ramiaquista) [reply src/redux/cofigureStore.js]: Thanks, @helman101, I've forgotten to import that, my bad. I will do it now!

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy:  . There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**